specification/resources/resource-manager/Microsoft.Resources/stable/2019-07-01/resources.json

```diff
@@ -601,6 +601,68 @@
         }
       }
     },
+    "/providers/Microsoft.Resources/deployments/{deploymentName}/whatIf": {
```

Comment:
>whatIf [](start = 65, length = 6)

Tenant and MG what-if only works in 2019-10-01 and later API versions, so I think we can drop the changes to 2019-07-01 and 2019-08-01.

Reply:
Got it. I'll restore 2019-07-01 and 2019-08-01.

Reply:
Good to know. Will restore 2019-07-01 and 2019-08-01.

Reply:
Done.